lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -2772,11 +2772,11 @@ def _get_additional_minimal_versions(self):
         self.option_minimal_versions['stop_timeout']['supported'] = stop_timeout_supported
 
     def __init__(self, **kwargs):
-        def detect_ipvX_address_usage():
+        def detect_ipvX_address_usage(client):
```

Comment:
I guess we can now move that function back out of `__init__()`, since there's no more reliance on `self`.